server/tests/search_providers/belga_360_archive_test.py

```diff
@@ -102,7 +102,7 @@ def test_format_list_item(self):
         ))
         self.assertEqual(item['body_html'], (
             '&nbsp;&nbsp;&nbsp;&nbsp;'
-            'Morbi lacus ex, molestie id ullamcorper quis, scelerisque quis lectus.<br/>&nbsp;&nbsp;&nbsp;&nbsp;'
+            'Morbi lacus ex, molestie id ullamcorper quis & scelerisque quis lectus.<br/>&nbsp;&nbsp;&nbsp;&nbsp;'
```

Comment:
shouldn't it be `&amp;` here?

Reply:
Nope, `&amp;` parse into `&` so here we will get `&`.

Reply:
so without that there would be no `&` ? I wonder how it would fail

Reply:
yep, imo in parse_html here https://github.com/superdesk/superdesk-core/blob/develop/superdesk/etree.py#L113 after when we convert it to string then somehow it missed.

Reply:
I see, but still shouldn't it be rather escaped like the other entities?

Reply:
imo I think it is better as we have to preserve ampersand because like in the string we have `&` and internally we are converting `&` into `&amp;` then in response we are getting again `&`, so simply passing ampersand **&** and getting in response **&**.

Reply:
ok, so can you maybe just test it with that co&v to be sure it works there?

Reply:
yep, I have tested, but if you say so in the test case instead of testing just **&** i will test with **co&v**.